go/vt/vttablet/onlineddl/executor.go

```diff
@@ -896,7 +896,10 @@ func (e *Executor) cutOverVReplMigration(ctx context.Context, s *VReplStream, sh
 		defer cancel()
 		// Wait for target to reach the up-to-date pos
 		if err := tmClient.VReplicationWaitForPos(ctx, tablet.Tablet, s.id, replication.EncodePosition(pos)); err != nil {
-			return err
+			if s, _ := e.readVReplStream(ctx, s.workflow, true); s != nil {
```

Comment:
As added bonus, why not also report what position we _did_ land at.

Reply:
This is also in the error message from the VReplication Engine FWIW.